packages/ERTP/test/unitTests/test-mintObj.js

```diff
@@ -104,3 +104,60 @@ test('mint.mintPayment inviteMathHelpers', t => {
     t.end();
   }
 });
+
+
+// Tests related to non-fungible tokens
+// This test models ballet tickets
+test('non-fungible tokens example', t => {
+  try {
+    const { mint: balletTicketMint, issuer: balletTicketIssuer, amountMath } = produceIssuer('Agoric Ballet Opera tickets', 'handle');
+
+    const startDateString = (new Date(2020, 1, 17, 20, 30)).toISOString();
+
+    const ticketDescriptionObjects = Array(5).fill().map((_, i) => ({
+      seat: i+1,
+      show: 'The Sofa',
+      start: startDateString,
+    }))
```

Comment:
I really like this example. I think there's two ways we could do this. One way (let's call it Version 1) is have the bare minimum identifier as the extent (perhaps using NatList). Then, there would have to be some API or offline resource to hit with that identifier Nat to learn what it is that you actually have.

The other way (Version 2) is to put all of this necessary information in the extent. This is better because in Version 1, we are dependent on another entity to determine the meaning of our digital assets, and they might misbehave. But, it's also more verbose, so we might be needing a lot more on-chain storage which may become a key factor later on.

It'd be great to have both of these options written out for Zoe Alpha, but I'm thinking that we might not have enough time for both. So, I think we should try Version 2 first. We will need to create a new MathHelpers that is based on the inviteMathHelpers, except without the instanceHandle. Perhaps we should drop the requirement to have an instanceHandle for inviteMathHelpers and use a renamed version inviteMathHelpers for both invites and these ballet tickets.

Reply:
I think this is outdated after the last discussion and #523 is the reflection of the current agreement
Please reopen if i missed something